products/sle/main.pm

```diff
@@ -703,8 +697,8 @@ elsif (get_var("WICKED")) {
     load_wicked_tests();
 }
 elsif (get_var("NFV")) {
-    boot_hdd_image();
-    load_nfv_tests();
+    load_baremetal_tests();
+    load_nfv_tests;
```

Comment:
sorry I thought it was clear that message is about BOTH lines :)

Reply:
ok, but there are many other load_text_x lines that don't have brackets

Reply:
sure , and I will keep fighting to fix all of them as much as I can :)

Reply:
and in this specific case you not changing this line at all you just *removing* brackets which were before. So no doubt it is not needed change